internal/controllers/clusterextension_controller_test.go

```diff
@@ -244,6 +172,11 @@ func TestClusterExtensionUnpackFails(t *testing.T) {
 	require.Equal(t, metav1.ConditionFalse, unpackedCond.Status)
 	require.Equal(t, ocv1alpha1.ReasonFailed, unpackedCond.Reason)
 
+	progressingCond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeProgressing)
+	require.NotNil(t, progressingCond)
+	require.Equal(t, metav1.ConditionTrue, progressingCond.Status)
+	require.Equal(t, ocv1alpha1.ReasonRetrying, progressingCond.Reason)
+
```

Comment:
Since we expected the `Progressing` condition message to be the place that contains details of the resolution, and there will eventually be no other place with that information, I think we should have some test assertions that look for <name>, <version>, and maybe <resolvedImageRef> in the message.

Reply:
I think this is reasonable

Reply:
should be updated in the latest push